### PR TITLE
sanitize_repo.sh now replaces in .kitchen.yml

### PR DESCRIPTION
**Pull request by yarmiganosca**

### CODE REVIEW

File: sanitize_repo.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # This assumes the current directory is named something like optoro_*
 # as per cookbook convention.
 DIRNAME=${PWD##*/}
-for i in $(grep -rl optoro_skel * | grep -v .git | grep -v ${0##*/} ); do
+for i in $(grep -rl optoro_skel . | grep -v .git | grep -v ${0##*/} ); do
   echo "Replacing optoro_skel in ${i}"
   #sed -i -e "s/optoro_skel/$DIRNAME/g" "$i"
   ruby -pi -e "gsub('optoro_skel', '$DIRNAME')" $i
```
Comment (zadunn) [inline sanitize_repo.sh]: a bash script should be bash.

Comment (yarmiganosca) [inline sanitize_repo.sh]: I tested the thing in bash. It works. I don't know what you mean by that. Could you be more precise?

Comment (zadunn) [inline sanitize_repo.sh]: Bash is a language.  If you are writing a bash script, it should all be in
bash.

Comment (yarmiganosca) [inline sanitize_repo.sh]: Oh, you're talking about the ruby invocation. Yeah, I didn't do that.

**Comment by yarmiganosca** [comment]
jenkins, retest this please

**Comment by zadunn** [comment]
@yarmiganosca version bump needed.

**Comment by yarmiganosca** [comment]
@zadunn version bump where?

**Comment by zadunn** [comment]
@yarmiganosca  metadata.rb in the root.

**Comment by yarmiganosca** [comment]
cool

**Pull request merged**